stdlib/REPL/test/precompilation.jl

```diff
@@ -26,7 +26,10 @@ if !Sys.iswindows()
         std = readuntil(ptm, "julia>")
         # check for newlines instead of equality with "julia>" because color may be on
         occursin("\n", std) && @info "There was output before the julia prompt:\n$std"
-        sleep(1) # sometimes precompiles output just after prompt appears
+        write(ptm, "\n")  # another prompt
+        readuntil(ptm, "julia>")
```

Comment:
FWIW, reading and writing from the same task is a data race violation and may deadlock or have other bad results. There must be a separate task for reading or the behavior of writing is undefined